### PR TITLE
Update CrossAccount.md

### PR DESCRIPTION
**Pull request by kevinemoore**
# Docs Update

Adding instructions to enable subscribing to existing object notifications (SNS topic) from a Quilt stack in another account.

# TODO
- [x] Documentation

### CODE REVIEW

File: docs/CrossAccount.md [hunk 1]
```diff
@@ -57,6 +57,31 @@ following to buckets in your *DataAccount*.
     ]
 }
 ```
+## Notifications
```
Comment (sir-sigurd) [inline docs/CrossAccount.md:60]:
```suggestion

## Notifications
```

File: docs/CrossAccount.md [hunk 2]
```diff
@@ -57,6 +57,31 @@ following to buckets in your *DataAccount*.
...
+      },
+      "Action": [
+        "sns:GetTopicAttributes",
+        "sns:Subscribe"
+      ],
+      "Resource": "SNS_TOPIC_ARN"
+}
+```
+
+
```
Comment (sir-sigurd) [inline docs/CrossAccount.md:82]:
```suggestion

```

File: docs/CrossAccount.md [hunk 3]
```diff
@@ -57,6 +57,31 @@ following to buckets in your *DataAccount*.
     ]
 }
 ```
+## Notifications
+
+In order for *ControlAccount* to use an existing and
+[correctly configured SNS topic](EventBridge.md#the-workarounds)
+for a single bucket in *DataAccount*, add a statement similar to the following
```
Comment (sir-sigurd) [inline docs/CrossAccount.md:64]: might make sense to emphasize that 'add' means to leave existing statements :blush:

File: docs/CrossAccount.md [hunk 4]
```diff
@@ -58,6 +58,30 @@ following to buckets in your *DataAccount*.
...
+        "AWS": "arn:aws:iam::CONTROL-ACCOUNT:root"
+      },
+      "Action": [
+        "sns:GetTopicAttributes",
+        "sns:Subscribe"
+      ],
+      "Resource": "SNS_TOPIC_ARN"
+}
+```
+You can now set the SNS topic in the [Catalog Admin Panel](catalog/Admin.md) in bucket
```
Comment (sir-sigurd) [inline docs/CrossAccount.md:82]:
```suggestion

You can now set the SNS topic in the [Catalog Admin Panel](catalog/Admin.md) in bucket
```

**Review by QuiltSimon — APPROVED** [review]
I dont totally love *ControlAccount* and *DataAccount* but this is good enough for now.

**Pull request merged**